db/migrate/20220610055050_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
Adding author_id in reference to the users table refers to the user_id. Remove the user_id column because you're making use of the author_id

Reply:
Done

Reply:
This issue is well resolved! ✔️